tests/test_harlequin.py

```diff
@@ -30,10 +31,16 @@ def test_non_existent_database(self):
         with pytest.raises(CommandError) as excinfo:
             call_command("--database", "nonexistent")
 
-        assert excinfo.value.args[0] == (
-            "Error: argument --database: invalid choice: 'nonexistent' "
-            + "(choose from 'default')"
-        )
+        if sys.version_info >= (3, 13, 1):
+            assert excinfo.value.args[0] == (
+                "Error: argument --database: invalid choice: 'nonexistent' "
+                + "(choose from default)"
+            )
+        else:
+            assert excinfo.value.args[0] == (
+                "Error: argument --database: invalid choice: 'nonexistent' "
+                + "(choose from 'default')"
+            )
```

Comment:
Hi Adam, I am a big fan of your work and can't help read your code whenever I get a chance to help me grow. I am unable to understand this bit here, why you introduced the version check if the outcome is the same regardless. I would love if you could explain. On another note, thank you greatly for your contribution to the Pytho community, especially to Django, and hope to see you on the Steering Council again some day. Cheers!